src/redux/reducers/createArticleReducer.js

```diff
@@ -1,58 +1,63 @@
 import {
   NEW_ARTICLE,
-  INPUT_CHANGE,
   ARTICLE_ERROR,
   SUBMITTING_ARTICLE,
   FETCH_ARTICLE_TO_EDIT,
   ARTICLE_UPDATED,
-  NEW_TAG,
-  REMOVE_TAG
+  CLEAR_RESPONSE
 } from "../actionTypes";
 
 const initialState = {
-  title: "",
-  body: "",
-  description: "",
-  tagsList: []
+  article: {
+    title: "",
+    body: "",
+    description: "",
+    tagsList: []
+  }
 };
 
 const createArticleReducer = (state = initialState, action) => {
   switch (action.type) {
     case NEW_ARTICLE:
       return {
         ...state,
-        response: action.payload
+        article: { ...state.article, ...action.payload.article },
+        message: action.payload.message,
+        response: action.payload.article,
+        isSubmitting: false
       };
     case ARTICLE_UPDATED:
       return {
         ...state,
-        response: action.payload
+        article: { ...state.article, ...action.payload.article },
+        message: action.payload.message,
+        response: action.payload.article,
+        isSubmitting: false
       };
-    case INPUT_CHANGE:
-      return { ...state, [action.payload.field]: action.payload.value };
+
     case ARTICLE_ERROR:
-      return { ...state, article_error: action.payload, isSubmitting: false };
-    case SUBMITTING_ARTICLE:
       return {
         ...state,
-        isSubmitting: true
+        articleError: action.payload,
+        isSubmitting: false
       };
-    case NEW_TAG:
+    case SUBMITTING_ARTICLE:
       return {
         ...state,
-        tagsList: [...state.tagsList, action.payload]
+        isSubmitting: true
       };
-    case REMOVE_TAG:
+    case CLEAR_RESPONSE:
```

Comment:
I look like this line is not covered in unit tests.
Can you consider covering it?